app/src/main/java/org/dslul/openboard/inputmethod/latin/Suggest.java

```diff
@@ -263,6 +266,20 @@ private void getSuggestedWordsForNonBatchInput(final WordComposer wordComposer,
         if (!TextUtils.isEmpty(typedWordString)) {
             suggestionsContainer.add(0, typedWordInfo);
         }
+		
+		if (!TextUtils.isEmpty(typedWordString)) {
+            List<String> results = EmojiSearch.getInstance().searchExact(typedWordString);
+            if (!results.isEmpty()) {
+                for (int idx = 0; idx < Math.min(2, results.size()); idx++) {
+                    suggestionsContainer.add(idx + 1, new SuggestedWordInfo(results.get(idx),
```

Comment:
A hacky solution to the issue mentioned in my comment is to insert emojis after the auto-correct entry. So:
```
suggestionsContainer.add(SuggestedWords.INDEX_OF_AUTO_CORRECTION + idx + 1, new SuggestedWordInfo(results.get(idx),
```

Reply:
Add less than three (**1-2**) emoji inside suggestions line, please!

(...correction word) Emoji, correction word, emoji (correction word...) **?**
--->  ` | ` :arrow_lower_left: ` | ` Arrow ` | ` :arrow_lower_right: ` | `